From df6917dd80dbcf1d0858e995ed867ce7cd59514b Mon Sep 17 00:00:00 2001
From: Raj Rajasekaran <b10872@freescale.com>
Date: Tue, 5 Aug 2008 17:05:13 -0500
Subject: [PATCH 040/435] ENGR00086696: RNG Driver unit test failed to build.

Fixed the Makefile to look for RNG Library under rootfs/usr/lib.

Signed-off-by: Raj Rajasekaran <b10872@freescale.com>
---
 test/mxc_rng_test/Makefile | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/test/mxc_rng_test/Makefile b/test/mxc_rng_test/Makefile
index 12b64d7..81e5f5c 100644
--- a/test/mxc_rng_test/Makefile
+++ b/test/mxc_rng_test/Makefile
@@ -51,9 +51,6 @@ TEST_SOURCES = apitest.c  run_tests.c  sym.c  hash.c  gen_encrypt.c         \
                callback.c  auth_decrypt.c  rand.c  hmac1.c  hmac2.c  cap.c  \
                wrap.c  results.c apihelp.c
 
-LIBS_DIR=$(TOPDIR)/lib/rng
-LIBS=$(LIBS_DIR)/libfsl_shw.a
-
 OPTFLAGS = -O2
 CFLAGS += $(OPTFLAGS) -DARM_LINUX -Wall $(INCLUDEDIR)
 
@@ -68,13 +65,10 @@ LD=$(CROSS_COMPILE)ld
 all: $(OBJS)
 
 rng_test: rng_test.o apihelp.o $(LIBS)
-	$(CC) -g -o $@ $^
+	$(CC) -lfsl_shw -g -o $@ $^
 
 apitest:  $(TEST_SOURCES:.c=.o) $(LIBS)
-	$(CC) -g -o $@ $^
-
-$(LIBS):
-	$(MAKE) -C $(LIBS_DIR)
+	$(CC) -lfsl_shw -g -o $@ $^
 
 move:
 	mkdir -p $(OBJDIR)
-- 
1.8.0

